mkosi/initrd.py

```diff
@@ -141,12 +142,32 @@ def initrd_finalize(staging_dir: str, output: str, output_dir: str) -> None:
     else:
         output_dir = os.fspath(Path.cwd())
 
+    # backup current image
+    if Path(f"{output_dir}/{output}").exists():
```

Comment:
The `Path` calls everywhere are quite ugly and the same string is made a `Path` many times over, maybe just make all of them `Path`s in the beginning.

Reply:
You're quite right...